### DISCUSSION TITLE
Issue importing Devices from CSV - v3.0.2

### DISCUSSION THREAD

**Discussion by chrisclarke6710** [netbox-community/netbox#7291] [Q&A] [answered]
Firstly - great project.

Version:  v3.0.2
I'm having an issue importing devices via CSV, both via a file and via CSV Data form.
Errors with:  
Row 1 device_role: Object not found
Row 1 device_type: Object not found
Row 1 site: Object not found

example csv: 
name,device_role,manufacturer,device_type,status,site
esx01,esxhost, Dell, r630, active, bn1

Any assistance appriciated.
thanks

**Comment by candlerb** [comment]
You need to pre-create the device role "esxhost", and if you have not already done so, pre-create manufacturer "Dell" and device type "r630".  Also you need to get rid of the spurious spaces after the commas.

**Reply by chrisclarke6710** [reply]
Many thanks for response.
I suspected that would be the case, therefore device role, manufacturer and device type were all pre-created. 
I've also checked for spaces around the commas and all fine.
But still have the same issue.

Thanks

**Reply by candlerb** [reply]
Does capitalisation match exactly?

**Comment by chrisclarke6710** [comment, marked as answer]
Apologies for late reply.
yes case matches however I think it may have been a browser issue.
Updated chrome and its working.

Many thanks
